test/scapy/layers/ntp.uts

```diff
@@ -1112,29 +1112,36 @@ pkt_1 = NTP(
     precision=RawVal(precision),
     dispersion=RawVal(dispersion),
     orig=RawVal(time_stamp),
-    sent=RawVal(time_stamp),
+    sent=RawVal(time_stamp)
```

Comment:
Please avoid making useless / unrelated changes as they tend to make history / git blame a bit more confusing
```suggestion
    sent=RawVal(time_stamp),
```